src/source.c

```diff
@@ -1858,6 +1858,43 @@ _dispatch_timers_get_delay(uint64_t nows[], struct dispatch_timer_s timer[],
 	return ridx;
 }
 
+
+#ifdef __linux__ 
+// in linux we map the _dispatch_kevent_qos_s  to struct kevent instead of struct kevent64.
+// we loose the kevent.ext[] members and the time out is based on relavite msec based time
+// vs. absolute nsec based time. For now we make the adjustments right here until the solution
+// to either extend libkqueue with a proper kevent64 API or removing kevent all together 
+// and move to a lower API (e.g. epoll or kernel_module. Also leeway is ignored.
+
+static void 
+kevent_set_delay(_dispatch_kevent_qos_s *ke, uint64_t delay,uint64_t leeway, uint64_t nows[]) 
```

Comment:
This is about timer so should have timer in the name.
also to match the naming style of the file this function should be

`_dispatch_kevent_timer_set_delay`.
please have a space after the commas, and wrap the prototype if it doesn't fit on 80 colums